ValidationMixin.js

```diff
@@ -44,18 +43,40 @@ var ValidationMixin = {
    * Convenience method to validate a key via an event handler. Useful for
    * onBlur, onClick, onChange, etc...
    *
-   * @param {?String} State key to validate
+   * Usage: <input onBlur={this.handleUnfocusFor('password')} .../>
+   *
+   * @param {?String} key to validate
    * default is false.
    * @return {function} validation event handler
    */
-  handleValidation: function(key) {
-    return function(event) {
+  handleUnfocusFor: function(key) {
+    return function handleUnfocus(event) {
       event.preventDefault();
       this.validate(key);
     }.bind(this);
   },
 
   /**
+   * Convenience method to validate a whole form on submit
+   *
+   * Usage: <form onSubmit={this.handleSubmit}>...</form>
+   */
+  handleSubmit: function(event) {
+    event.preventDefault();
+    this.validate();
+  },
+
+  /**
+   * Convenience method to reset a form to initial state
+   *
+   * Usage: <button onClick={this.handleReset}>Reset</button>
+   */
+  handleReset: function(event) {
```

Comment:
`handleReset` is a good suggestion for this library. I think I'll add `clearValidations` to the API that will simply clear out `this.state.errors`